packages/babel-preset-env/src/use-built-ins-plugin.js

```diff
@@ -119,22 +119,28 @@ export default function({ types: t }: { types: Object }): Plugin {
         return;
       }
 
-      addImport(path, "web.dom.iterable", this.builtIns);
+      addImport(path, "es.array.iterator", this.builtIns);
+      addImport(path, "es.string.iterator", this.builtIns);
+      addImport(path, "web.dom-collections.iterator", this.builtIns);
```

Comment:
Is this really required if we have those dependencies for `iterator` static property of `Symbol` in `built-in-definitions.js`?

Reply:
I don't think it is... iirc, @hzoo added this as just a general safety